app/controllers/categories_controller.rb

```diff
@@ -32,7 +29,6 @@ def create
     end
   end
 
-  # PATCH/PUT /categories/1 or /categories/1.json
   def update
```

Comment:
- [Optional] You are not making use of the update and destroy actions, let's better get rid of it.